### ISSUE TITLE
[Design, FE] 10-10EZ Short-from: Add unique name for Insurance Information edit buttons > [Link and button labels] Button and link language isn't descriptive and/or their purpose isn't clear. (08.03.3)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va-forms-system-core#357]
#### VFS team name
10-10 Team

#### VFS product name
Health Care Application 10-10EZ

#### VFS feature name
10-10EZ Short Form (Auto-qualifiers)

#### Point of Contact/Reviewers
Brian DeConinck (@briandeconinck) - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Button and link language isn't descriptive and/or their purpose isn't clear.

### Issue Details
On Step 2 of 3 [disability rating >50%]: Insurance Information, when a user has health insurance coverage and is adding multiple insurance policies, an Edit button is paired with each policy. Each of these Edit buttons have identical accessible names, "Edit." For screen reader users tabbing through the buttons, it's difficult to identify which policy will be edited by following which button.

### Link, screenshot or steps to recreate
1. Log into the RI as user 228.
2. Complete the form up to Step 2 of 3.
3. For "Do you have health insurance coverage?" answer Yes.
4. Enter information for an insurance policy.
5. Click "Add another insurance policy."
6. Enter information for another insurance policy.
7. Click "Add another insurance policy."
8. Activate a screen reader (recommend NVDA on Windows or VoiceOver on Mac for testing).
9. Press [tab] or [shift] + [tab] to move forward and backward through the interactive elements on the page.
10. Note that the edit buttons for each insurance policy are announced only as "Edit."

### VA.gov Experience Standard
[Category Number 08, Issue Number 03](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.4_A

---

### Platform Recommendation
Each edit button should have a unique accessible name. This can be accomplished either by changing the button text (`<button>Edit [name of policy]</button>`) or by providing a unique aria-label (`<button aria-label="Edit [name of policy]">Edit</button>`).

### VFS Team Tasks to Complete

- [x] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by micahchiang** [comment]
Update: @obliviga has a [pull request](https://github.com/department-of-veterans-affairs/vets-website/pull/21312) out that will fix this **specifically** for the 10-10EZ. We discovered the functionality for unique aria-labels already exists, but wasn't documented anywhere. We'll be updating our documentation to include an entry for `itemAriaLabel`, which accepts a function that takes formdata as it's parameter and can return the name for the individual entry.

**Comment by obliviga** [comment]
Verified that this has been deployed to prod:
![image.png](https://images.zenhubusercontent.com/6283eaffdea4c21364d67e5a/40b16e47-4283-41aa-959b-85e5ba8809b7)

**Comment by hdjustice** [comment]
This looks awesome!
